examples/performance-sample/build.gradle

```diff
@@ -417,3 +433,110 @@ class TimingsListener implements TaskExecutionListener, BuildListener {
 if (project.hasProperty('profile')) {
   gradle.addListener new TimingsListener()
 }
+
+
+import com.marklogic.client.DatabaseClient
+import com.marklogic.client.io.FileHandle
+import com.marklogic.contentpump.bean.MlcpBean
+import org.gradle.api.logging.Logger
+import org.gradle.api.logging.Logging
+import org.gradle.api.tasks.JavaExec
+import org.gradle.api.tasks.TaskAction
+
+import com.marklogic.appdeployer.AppConfig
+class MlcpTask extends JavaExec {

```

Comment:
isn't this part of ml-gradle? from [Rob's wiki](https://github.com/marklogic-community/ml-gradle/wiki/Content-Pump-and-Gradle) I wouldn't think this would be needed (but I'm still learning about gradle)

Reply:
he hasn't updated to the latest one yet. I forgot to put a comment in there about it. will do that.